src/JWT/Jwk/IJwtWebKeysCollectionFactory.cs

```diff
@@ -0,0 +1,6 @@
+namespace JWT.Jwk;
```

Comment:
Please convert implicit namespace into explicit (here and elsewhere too), for the consistency of the codebase

Reply:
Oops. sorry. This is the only place with implicit namespace (my bad, miss it). Fixed